amd64/Dockerfile

```diff
@@ -38,7 +40,9 @@ RUN apt-get update && \
         sqlite3 \
         tigervnc-standalone-server \
         tigervnc-common \
-        wmii \
+	novnc \
+	websockify \
+        openbox \
```

Comment:
Is the tabbing messed up here? Please correct the indentation of novnc and websockify if so.